website/docs/registry/faq.mdx

```diff
@@ -0,0 +1,49 @@
+---
+
+## Common publishing issues
+
+### How does the Registry identify the latest version of my provider, module, or policy library?
+
+Each version of an integration published in the Terraform Registry has a version identifier, which is a string like `1.2.3`. You create the version identifier when you create a tag (for modules and policy libraries) or a release (for providers) in the GitHub repository where the integration's code lives.
+
+The Registry requires that these version identifiers are [SemVer compliant](https://semver.org/). Versions are sorted using these identifiers, following [standard SemVer rules for precedence](https://semver.org/#spec-item-11). The version identifier with the greatest precedence is always listed as latest by the Terraform Registry.
+
+### Why are prerelease versions listed as the latest version of my integration?
+
+The Terraform Registry's version sorting is strictly SemVer compliant (see above). This means that "latest" in the Registry can be a prerelease version, if the prerelease's version identifier has the greatest precedence.
+
+For example, if your provider has a published version 1 (i.e. `1.0.0`), and then you create a new release for a version 2 prerelease (e.g. `2.0.0-preview`), the version 2 prerelease will be listed as the latest version in the Terraform Registry.
+
+### How do I remove a version of my provider, module, or policy library that has already been published?
+
+If any users have already begun using a version of your provider, module, or policy library, removing the version from the Terraform Registry will break their Terraform configuration. They may be unable to work with their infrastructure until they can find a suitable replacement version of your integration. For this reason, **we recommend that you never remove previously published versions of providers, modules, or policy libraries**.
+
+If there is a bug in a certain version, it is usually better to publish another release, incrementing the patch version (i.e. from `1.2.0` to `1.2.1`), rather than to remove the bugged version.
+
+There may be situations where it is important that a version be removed from the Registry. Perhaps a version of a provider has a critical security vulnerability, or you may have extremely high confidence that no one is using a version of your module that doesn't work as you intended it to.
+
+In these situations, your options for removing previously published versions vary depending on the type of integration:
+
+- **Modules**: The owner of a module can use the Terraform Registry website to remove a specific version, all versions of a specific module provider, or remove the module entirely. [More details on this workflow can be found here](/registry/modules/publish#removing-published-content). or [here for policy libraries].
+- **Providers**: There is no manual process for removing a version. Reach out to us via email if it is critical that a version be deleted.
```

Comment:
Might want to include that we will take the request under consideration, but it's not guaranteed we will remove it. Not sure if there are GDPR implications to saying something like that though.